app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,226 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.NewTask;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스")
+class TaskServiceTest {
+
+    private TaskService taskService;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks {
+        List<Task> subject() {
+            return taskService.getTasks();
+        }
+
+        @Nested
+        @DisplayName("DB에 데이터가 존재하지 않는다면")
+        class Context_with_empty_db {
+            @Test
+            @DisplayName("빈 ArrayList를 리턴한다")
```

Comment:
`getTasks`의 리턴 타입이 `List<Task>` 였던 것 같습니다. `ArrayList`를 리턴하는 것은 구현에 대해서는 사실일 거에요. 그러나 `getTasks`의 메소드 시그니처에 대해서는 중요한 정보가 아닙니다.

```suggestion
            @DisplayName("빈 List를 리턴한다")
```